Sources/NIOCore/AsyncSequences/NIOThrowingAsyncSequenceProducer.swift

```diff
@@ -870,7 +870,7 @@ extension NIOThrowingAsyncSequenceProducer {
             case callDidTerminate
             /// Indicates that the continuation should be resumed with `nil` and
             /// that ``NIOAsyncSequenceProducerDelegate/didTerminate()`` should be called.
-            case resumeContinuationWithNilAndCallDidTerminate(CheckedContinuation<Element?, Error>)
+            case resumeContinuationWithCancellationErrorAndCallDidTerminate(CheckedContinuation<Element?, Error>)
```

Comment:
Can you update the comment above?

Reply:
Changed in https://github.com/apple/swift-nio/pull/2399/commits/1e41b2bf459f73ba70b5d0608b8267a34009b638